examples/Cargo.toml

```diff
@@ -33,13 +33,10 @@ anyhow = { version = "1", features = ["backtrace"] }
 
 
 [target.'cfg(target_os="espidf")'.dependencies]
-esp-idf-sys = { version = "0.33.7", features = ["binstart"] }
 anyhow = {version = "1", features = ["backtrace"]}
 embedded-svc = "0.26.4"
 embedded-hal = { version = "0.2.7", features = ["unproven"]}
 micro-rdk = { path = "../", default-features = false, features = ["esp32"] }
-esp-idf-svc = { version = "0.47.3", features = ["experimental"] }
```

Comment:
Note: this removal breaks building the examples, and I am not sure why. The error is:
```
  = note: [ldproxy] Running ldproxy
          thread 'main' panicked at /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/ldproxy-0.3.3/src/main.rs:44:13:
          Cannot locate argument '--ldproxy-linker <linker>'
```
I'm investigating.

Reply:
Pre above, I'm going to back out this removal for now.